src/content/pages/programme/rust-summit.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Rust Summit
+subtitle: The Rust Summit aims to bring together various stakeholders of Python’s Rust ecosystem to discuss its current state, address challenges, and align on ongoing work.
+
+---
+## **Rust Summit**
+
+The **Rust Summit** aims to bring together various stakeholders of Python’s Rust ecosystem to discuss its current state, address challenges, and align on ongoing work.
+
+**WHEN?** The summit will take place on **Monday, July 14th** or **Tuesday, July 15th** (confirmation pending).
+
+**WHERE?** The event will be hosted at the **Prague Congress Centre (PCC)**, Room \[TBA\].
+
+---
+
+#### **Meet the Community**
+
+#### The main goal of this summit is for various stakeholders of Python’s Rust ecosystem to meet, learn, and discuss about the Rust ecosystem in Python, how Python can benefit from Rust, and anything related to the adaptation of thread-free Python.
```

Comment:
Maybe change this from a heading into a regular paragraph? 
https://rust-summit.ep-preview.click/programme/rust-summit/

```suggestion
The main goal of this summit is for various stakeholders of Python’s Rust ecosystem to meet, learn, and discuss about the Rust ecosystem in Python, how Python can benefit from Rust, and anything related to the adaptation of thread-free Python.
```

Reply:
I would like to accept your suggestion but it is not my PR anymore, @rodrigogiraoserrao could you please help?

Reply:
NVM @rodrigogiraoserrao  i realized I can commit the suggestion, thanks!